### DISCUSSION TITLE
Add more shades of gray

### DISCUSSION THREAD

**Discussion by yayvery** [tailwindlabs/tailwindcss#3569] [Ideas]
The current gray palette does not have enough options to make a good looking dark theme. I needed to add `gray-650`, `gray-750`, `gray-850` and `gray-950`. I calculated the colors using [this color blender website](https://meyerweb.com/eric/tools/color-blend/#:::hex). These shades should be included by default.

**Comment by marcia-lima** [comment]
The following discussions might be related to yours:

- [#3081](https://github.com/tailwindlabs/tailwindcss/discussions/3081)
- [#4130](https://github.com/tailwindlabs/tailwindcss/discussions/4130)
